.circleci/config.yml

```diff
@@ -1,18 +1,25 @@
 version: 2
 
-jobs:
-  build:
+references:
```

Comment:
I started moving some of our shared commands into `references`. Not sure how far we want to take it, but thought this PR could be a good proof-of-concept